backend/src/ml_space_lambda/data_access_objects/config_profiles.py

```diff
@@ -0,0 +1,179 @@
+import json
+import logging
+import time
+import uuid
+from typing import List, Optional, Dict, Any
+
+import boto3
+from dynamodb_json import json_util as dynamodb_json
+from boto3.dynamodb.conditions import Key
+
+from ml_space_lambda.data_access_objects.dynamo_data_store import DynamoDBObjectStore
+from ml_space_lambda.enums import EnvVariable
+from ml_space_lambda.utils.exceptions import ResourceInUseError
+from ml_space_lambda.utils.mlspace_config import get_environment_variables
+
+log = logging.getLogger(__name__)
+
+
+class ConfigProfileModel:
```

Comment:
Not something you have to do since it isn't used throughout this project, but we'd really benefit from using pydantic's BaseModel instead of doing all the init and to_dict/from_dict by hand. You can [look at LISA](https://github.com/awslabs/LISA/blob/main/lambda/models/domain_objects.py#L420-L432) to see how simple it can be for the same functionality.